pkg/cmd/pr/create/create_test.go

```diff
@@ -908,17 +855,10 @@ func Test_determineTrackingBranch_respectTrackingConfig(t *testing.T) {
 		},
 	}
 
-	cs.Stub(`branch.feature.remote origin
-branch.feature.merge refs/heads/great-feat`) // git config --get-regexp (ReadBranchConfig)
```

Comment:
The contents of this stub was replaced with an empty string in the rewrite. I will make a change to bring back the original value